marine/marine_pool.py

```diff
@@ -0,0 +1,52 @@
+import itertools
+import math
+import multiprocessing
+from dataclasses import dataclass
+from functools import partial
+from typing import List, Dict, Optional, Tuple
+
+from marine import Marine
+
+
+@dataclass
+class MarinePool:
+    _lib_path: str
+    process_count: int = 4
+
+    def filter_and_parse(
+        self,
+        packets: List[bytes],
+        bpf: Optional[str] = None,
+        display_filter: Optional[str] = None,
+        fields: Optional[List[str]] = None,
+    ) -> List[Tuple[bool, Dict[str, str]]]:
+        if len(packets) == 0:
+            return []
+
+        ctx = multiprocessing.get_context("spawn")
```

Comment:
Why not just use `pool = multiprocessing.Pool(count)`?

Reply:
The default fork_method is "fork" and not "spawn". I assumed @tomlegkov chose spawn for a reason...

Reply:
Yeah, he explained it on slack.
Add a comment explaining it so some wise-ass won't change it in the future without understanding.